scripts/ssl-common.sh

```diff
@@ -7,13 +7,15 @@ if [ -z "${easyrsa_bin-}" ] || [ ! -x "${easyrsa_bin}" ]; then
     if [ -z "${easyrsa_bin}" ]; then
         easyrsa_dir="$(mktemp -dt easyrsa.XXXXXXXX)"
         easyrsa_url="https://github.com/OpenVPN/easy-rsa/releases/download/v3.0.5/EasyRSA-nix-3.0.5.tgz"
+        echo "  - Downloading EasyRSA..."
```

Comment:
I'd say these are unnecessary, the point of `ssl-common` was to keep it transparent from the user.

Reply:
the delay made it unclear what was happening - I noticed it and was initially concerned something wasn't working as intended, so this log output makes it clear we are pending a download operation.